plugins/justify/plugin.js

```diff
@@ -150,6 +149,12 @@
 					if ( block.isReadOnly() )
 						continue;
 
+					// #16808 Check if style or class might be applied to currently processed element
+					var tag = block.getName(),
+						isAllowedTextAlign, isAllowedCssClass;
+					isAllowedTextAlign = editor.filter.check( tag + '{text-align}' ) || false;
+					isAllowedCssClass = editor.filter.check( tag + '(' + cssClassName + ')' ) || false;
```

Comment:
I made a condition to continue loop if both condition are false. You have right, default values are not necessary `filter.check` return bool.  I updated also `check` method to use `activeFilter`.